src/pdn/src/grid.cpp

```diff
@@ -600,8 +600,7 @@ void Grid::makeVias(const ShapeTreeMap& global_shapes,
   debugPrint(getLogger(), utl::PDN, "Make", 1, "Making vias in \"{}\"", name_);
   ShapeTreeMap search_shapes = getShapes();
 
-  odb::Rect search_area;
-  search_area.mergeInit();
+  odb::Rect search_area = getGridBoundary();
   for (const auto& [layer, shapes] : search_shapes) {
     for (const auto& [box, shape] : shapes) {
       search_area.merge(shape->getRect());
```

Comment:
If you are using the grid boundary, is there any need to merge in the shapes?  Wouldn't they always be inside the grid boundary?